### DISCUSSION TITLE
How do I write data to opc?

### DISCUSSION THREAD

**Discussion by yaron3** [gopcua/opcua#447] [Q&A]
I am looking how to write to opc server.
I could read using: attrs, err := n.Attributes(ua.AttributeIDBrowseName, ua.AttributeIDDataType,ua.AttributeIDValue)
and then read the value with: attrs[2].Value.Value()

how do I do the opposite way?

**Comment by magiconair** [comment]
Try the `Write` request on the client. https://github.com/gopcua/opcua/blob/main/client.go#L815-L822
You can find an example in the `examples/write` folder.

**Comment by magiconair** [comment]
Try the `Write` request on the client. https://github.com/gopcua/opcua/blob/main/client.go#L815-L822
You can find an example in the `examples/write` folder.
